conda/environments/cudf_dev_cuda11.5.yml

```
@@ -67,6 +67,13 @@ dependencies:
   - pydata-sphinx-theme
   - librdkafka=1.7.0
   - python-confluent-kafka=1.7.0
+  - moto>=3.1.6
+  - boto3>=1.21.21
+  - botocore>=1.24.21
+  - aiobotocore>=2.2.0
+  - s3fs>=2022.3.0
+  - flask
```

Could we add a minimum major version pin? There are multiple major versions of flask in the wild with API breaks that would affect compatibility.
```suggestion
  - flask>=2.0.0
```

---

Removed this dependency since it was added prior to upgrading the `moto` pin. Older versions of `moto` required `flask` & `flask_cors` to be installed, the latest version pinning for `moto` solves this issue though.